### PR TITLE
1887 het to dol

### PR DESCRIPTION
**Pull request by tillmo**
change extension .het to .dol

### CODE REVIEW

File: ExtModal/Tries/Makefile [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 .PHONY: check
 
-check: hets
-	../../hets -v2 -o pp.het,th Try?.het
```
Comment (mcodescu) [inline ExtModal/Tries/Makefile]: it should stay `hets -v2 -o pp.dol`

File: Fpl/test/Makefile [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 .PHONY: check
 
-check: hets
-	../../hets -v2 -o pp.het,th specs.het
-	../../hets -v2 -o pp.het,th specs.pp.het
+check:.dols
```
Comment (mcodescu) [inline Fpl/test/Makefile]: should stay `hets -v2 -o pp.dol` and perhaps `specs.dol`

**Comment by mcodescu** [comment]
Otherwise there are some files in `utils` that use `.het`.

**Comment by tillmo** [comment]
> For backwards compatibility reasons, would it be possible to also support `.het` in the same way as `.dol`?

We still support .het as input format, but only pp.dol (not pp.het) as output format (supporting both would be more complicated)

**Comment by tillmo** [comment]
> Otherwise there are some files in `utils` that use `.het`.

I have now renamed all `.het` files into `.dol`. I could not find any `.het` file in `utils`.

**Comment by tillmo** [comment]
OK, tests go through, please review again

**Pull request merged**